README.rst

```diff
@@ -126,6 +126,33 @@ The containerized version of the CLI can be launched using Docker command.
 It makes sure your current directory is mounted to the same place in the
 container.
 
+
+Developing Renku
+================
+
+For development it's convenient to install ``renku`` in editable mode. This is
+still possible with ``pipx``. First clone the repository and then do:
+
+::
+
+    $ pipx install \
+        --editable \
+        --spec <path-to-renku-python>[all] \
+        renku
+
+This will install all the extras for testing and debugging.
+
+Debugging
+---------
+
+If you use Visual Studio Code, it is very convenient to debug ``renku`` using
```

Comment:
I would not add editor specific debugging information to project level. It's personal what you use and we should be neutral about that topic on the project level.